### ISSUE TITLE
Add Sylvester to benchmark

### ISSUE THREAD

**Issue by feklee** [stepheneb/webgl-matrix-benchmarks#21]
According to the [Numeric Javascript benchmark](http://www.numericjs.com/benchmark.html), Sylvester is faster than Google Closure for important "3D operations" such as 3x3 or 4x4 matrix multiplications, in Chrome 20.0 on WinXP/32. Therefore, Sylvester would be an interesting addition to the _WebGL Matrix Library Benchmark_.

**Comment by stepheneb** [comment]
I agree that would be nice to add https://github.com/jcoglan/sylvester

Would you like to try adding it ? 

If you fork this repo, make a new branch for adding sylvester and get it working and make a pull request I love to merge and deploy the change.

**Comment by feklee** [comment]
_Added and pull request sent._

With Chrome 20 on WinXP/32, performance of Sylvester indeed is very low compared to all other libraries in the benchmark. Furthermore, functionality dedicated to working with 4x4 matrices for transformations in 3D space is missing. Therefore not all tests are available.

Concerning the [Numeric Javascript benchmark](http://www.numericjs.com/benchmark.html): As far as I can tell, it tests Closure's generic matrix module. _webgl-matrix-benchmarks_, on the other hand, tests Closure modules dedicated to WebGL matrix transformations.

Possibly of interest: Although Sylvester is not a WebGL library, it is used in the example code of MDN's [Getting started with WebGL](https://developer.mozilla.org/en/WebGL/Getting_started_with_WebGL).
